recipes/python-certifi-win32/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "python-certifi-win32" %}
+{% set version = "1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e074663a1b6e54233f28d15832f9872440472144abafa432ec7f5d473fa33085
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - certifi
+    - python
+    - wincertstore
+    - wrapt >=1.10.4
+
+test:
+  imports:
+    - certifi_win32
+
+about:
+  home: "https://gitlab.com/alelec/python-certifi-win32"
+  license: "BSD 2-clause Simplified"
```

We use https://spdx.org/licenses/ for license names
```suggestion
  license: BSD-2-Clause
```